src/balena.sbvr

```diff
@@ -796,6 +802,8 @@ Fact type: release has asset key
 -- release asset
 Fact type: release asset has asset
 	Necessity: each release asset has at most one asset.
+	-- The content_type size is as per RFC6838 recommendation: "<type-name> and <subtype-name> SHOULD be limited to 64 characters."
+	Necessity: each release asset that has an asset, has an asset that has a Filename (Type) that has a Length (Type) that is less than or equal to 255 and has a Content Type (Type) that has a Length (Type) that is less than or equal to 129.
```

Comment:
These look fine to me. You might want to check with product if 255 is not too big (but I think it is fine as it is also kind of similar to what windows allows)